stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/MediaStringUtil.kt

```diff
@@ -17,27 +17,34 @@
 package com.getstream.sdk.chat.utils
 
 import java.text.DecimalFormat
+import java.util.Locale
 import kotlin.math.ln
 import kotlin.math.pow
 
+private const val SECONDS_IN_A_HOUR = 3600
+private const val SECONDS_IN_A_MINUTE = 60
+private const val BYTE_UNIT_CONVERSION_FACTOR = 1024
 public object MediaStringUtil {
 
     @JvmStatic
-    public fun convertVideoLength(videoLength: Long): String {
-        val hours = videoLength / 3600
-        val minutes = videoLength % 3600 / 60
-        val seconds = videoLength % 60
-        return String.format("%02d:%02d:%02d", hours, minutes, seconds)
+    public fun convertVideoLength(videoLengthInSeconds: Long, locale: Locale = Locale.getDefault()): String {
+        val hours = videoLengthInSeconds / SECONDS_IN_A_HOUR
+        val minutes = videoLengthInSeconds % SECONDS_IN_A_HOUR / SECONDS_IN_A_MINUTE
+        val seconds = videoLengthInSeconds % SECONDS_IN_A_MINUTE
+        return String.format(locale, "%02d:%02d:%02d", hours, minutes, seconds)
     }
 
     @JvmStatic
     public fun convertFileSizeByteCount(bytes: Long): String {
-        val unit = 1024
-        if (bytes <= 0) return 0.toString() + " B"
-        if (bytes < unit) return "$bytes B"
-        val exp = (ln(bytes.toDouble()) / ln(unit.toDouble())).toInt()
-        val pre = "KMGTPE"[exp - 1].toString()
-        val df = DecimalFormat("###.##")
-        return df.format(bytes / unit.toDouble().pow(exp.toDouble())) + " " + pre + "B"
+        return when {
+            bytes <= 0 -> 0.toString() + " B"
```

Comment:
We should probably use string interpolation ( "0 B") or just a string, rather than concatenation and toString on a number